FinniversKit/Sources/Recycling/GridViews/AdRecommendations/GridView/AdRecommendationsGridViewLayout.swift

```diff
@@ -122,7 +118,13 @@ class AdRecommendationsGridViewLayout: UICollectionViewLayout {
     }
 
     override func layoutAttributesForItem(at indexPath: IndexPath) -> UICollectionViewLayoutAttributes? {
-        return itemAttributes[indexPath.row]
+        guard itemAttributes.indices.contains(indexPath.row)
+        else { return nil }
+        if itemAttributes[indexPath.row].representedElementCategory == .cell {
```

Comment:
`layoutAttributesForItem ` retrieves layout info at the indexPath for cells only
we create above a `UICollectionViewLayoutAttributes` for the header which is stored in the same array

for some reason, on iPads when launching the app from background this method is called with indexPath.row == 0 which would cause a crash, since at index 0 in the array we have a layout attribute for a header and not for a cell
on iPhones, that does not happen 🤷‍♂️